venue/views.py

```diff
@@ -1190,6 +1202,15 @@ def confirm_email_change(request):
         token = AuthToken.objects.get(salt=code)
         new_email = rtemp.retrieve(code)
         if code and new_email:
+            # Update the email in Constant Contact
+            resp = get_contstant_contact_record(token.user.email)
+            if resp.status_code == 200 and len(resp.json()['results']) > 0:
```

Comment:
Does resp.json() actually do parsing? If so, it should be done only once and the result accessed twice, otherwise the parsing work is doubled.